amethyst_core/src/float.rs

```diff
@@ -29,17 +29,17 @@ type FloatBase = f32;
 #[derive(Alga, Clone, Copy, PartialOrd, PartialEq, Serialize, Deserialize, Debug)]
 #[alga_traits(Field(Additive, Multiplicative))]
 #[serde(transparent)]
-pub struct Float(FloatBase);
+pub struct f32(FloatBase);
```

Comment:
press x to doubt

Reply:
Abusing clion refactor to make renaming easier ;)

Reply:
Actually removed it now. Wasn't actually used anymore or even exported, just left it for reference for clion renaming.